README.md

```diff
@@ -1,2 +1,2 @@
 # form-auto-revieuw
-i will revieuw this code mt self
+i will revieuw this code my self
```

Comment:
describe well your work in order to make everything clear!!!

Reply:
@faustM it would be better to use suggestion words more than commands words like you should or it would be better please try to do that